Source/Lib/ASM_SSE4_1/EbTransforms_Intrinsic_SSE4_1.c

```diff
@@ -7,6 +7,213 @@
 #include <smmintrin.h>
 #include "EbTransforms_SSE4_1.h"
 
+EB_ALIGN(16) const EB_S16 TransformAsmConst_SSE4_1[] = {
```

Comment:
why the change here?

Reply:
I kept getting error for undefined with it, I'm going to try again with the commit reverted with the others fixed

Reply:
It seems I might be able to revert